package.json

```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^3.9.2",
     "axios": "^0.18.0",
     "g": "^2.0.1",
```

Comment:
What is g? Maybe we should remove it, if it's not a package a used package. I'll check in a bit.